server/services/sample_data/flights_saved_objects.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+import { ConfigSchema } from '../../../common/config';
+
+const layerList = (config: ConfigSchema) => [
```

Comment:
nit: can this be destructured?

Reply:
I tried. But, not able to destructure it.